### PR TITLE
Add SoCC22 paper

### PR DESCRIPTION
**Pull request by gaulthiergain**
Add link to SoCC'22 and slides.

Signed-off-by: Gaulthier Gain <gaulthier.gain@uliege.be>

### CODE REVIEW

File: data/papers.yaml [hunk 1]
```diff
@@ -1,3 +1,14 @@
+- title: "Want More Unikernels? Inflate Them!"
+  venue: ACM SoCC'22
+  year: 2022
+  pdf: https://dl.acm.org/doi/abs/10.1145/3542929.3563473
```
Comment (razvand) [inline data/papers.yaml:4]:
```suggestion
  link: https://dl.acm.org/doi/abs/10.1145/3542929.3563473
  pdf: https://dl.acm.org/doi/pdf/10.1145/3542929.3563473
```

**Review by razvand — CHANGES_REQUESTED** [review]
Thanks for this, @gaulthiergain . Please see my comment.

File: data/papers.yaml (hunk 1 above)
Comment (gaulthiergain) [reply data/papers.yaml:4]: Thanks @razvand. We can commit with your changes.

**Review by razvand — APPROVED** [review]
Thanks, @gaulthiergain 

Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**